app/editor/src/features/admin/sources/connections/AudioClip.tsx

```diff
@@ -70,6 +70,15 @@ export const AudioClip: React.FC = (props) => {
           setFieldValue('connection.throwOnMissingFile', e.currentTarget.checked);
         }}
       />
+      <p>Do not turn on "Import Content" until you have successfully ingested content.</p>
+      <FormikCheckbox
```

Comment:
I don't love this solution, but it functions for now.  Essentially, the Content Service will error out if the Kafka topic does not exist.  If an Ingest Service hasn't pushed any messages to the Kafka topic, then the topic will probably not exist (unless it was manually created).